app/models/match_decisions/base.rb

```diff
@@ -86,7 +86,7 @@ def contact_name
     def editable?
       # can this decision be updated by a notification response?
       # override this default behavior in subclasses
-      initialized? && match_open?
+      initialized? && match_open? && saved_status !~ /acknowledged|accepted|confirmed|declined|canceled|rejected|complete|completed|scheduled|no_hearing|mitigation_required|mitigation_not_required|decline_overridden|decline_overridden_returned|decline_confirmed/
```

Comment:
does this regex want anchors? Assuming we always want to match the entire status 
```suggestion
      initialized? && match_open? && saved_status !~ /\A(acknowledged|accepted|confirmed|declined|canceled|rejected|complete|completed|scheduled|no_hearing|mitigation_required|mitigation_not_required|decline_overridden|decline_overridden_returned|decline_confirmed)\z/
```